tests/unit/test_parameters/test_parameter_sets/test_LCO_Ai2020.py

```diff
@@ -0,0 +1,57 @@
+#
+# Tests for Ai (2020) Enertech parameter set loads
+#
+import pybamm
+import unittest
+
+
+class TestAi2020(unittest.TestCase):
+    def test_load_params(self):
+        anode = pybamm.ParameterValues({}).read_parameters_csv(
+            pybamm.get_parameters_filepath(
+                "input/parameters/lithium-ion/anodes/graphite_Ai2020/parameters.csv"
+            )
+        )
+        self.assertEqual(anode["Negative electrode porosity"], "0.33")
+
+        cathode = pybamm.ParameterValues({}).read_parameters_csv(
+            pybamm.get_parameters_filepath(
+                "input/parameters/lithium-ion/cathodes/lico2_Ai2020/parameters.csv"
+            )
+        )
+        self.assertEqual(cathode["Positive electrode porosity"], "0.32")
+
+        electrolyte = pybamm.ParameterValues({}).read_parameters_csv(
+            pybamm.get_parameters_filepath(
+                "input/parameters/lithium-ion/electrolytes/lipf6_Enertech_Ai2020/"
+                + "parameters.csv"
+            )
+        )
+        self.assertEqual(electrolyte["Cation transference number"], "0.38")
+
+        cell = pybamm.ParameterValues({}).read_parameters_csv(
+            pybamm.get_parameters_filepath(
+                "input/parameters/lithium-ion/cells/Enertech_Ai2020/parameters.csv"
+            )
+        )
+        self.assertAlmostEqual(cell["Negative current collector thickness [m]"], 10e-6)
+
+    def test_standard_lithium_parameters(self):
+
+        chemistry = pybamm.parameter_sets.Ai2020
+        parameter_values = pybamm.ParameterValues(chemistry=chemistry)
+        options = {"particle": "Fickian diffusion", "particle cracking": "anode"}
```

Comment:
I think we are almost there. To increase the coverage in the LiCo2 parameters you should include the `cathode` here, either in this model, or just copy the whole test and run it with cracking in the cathode. This will also improve the coverage in `base_lithium_ion_model.py`.